From b3c9254ee155e2b810a395e36a9f3ee9e7fb3b97 Mon Sep 17 00:00:00 2001
From: Quintin Hill <Quintin.Hill@arqiva.com>
Date: Wed, 12 Apr 2017 15:50:04 +0100
Subject: [PATCH 3/6] submit: allow whitespace in Output and Error paths

This was already allowed with the Executable and Log paths so consistency of
support is improved.
---
 src/condor_utils/submit_utils.cpp | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/src/condor_utils/submit_utils.cpp b/src/condor_utils/submit_utils.cpp
index 2dbd22b572..6e4d1eafb3 100644
--- a/src/condor_utils/submit_utils.cpp
+++ b/src/condor_utils/submit_utils.cpp
@@ -450,17 +450,6 @@ static char * trim_and_strip_quotes_in_place(char * str)
 	return p;
 }
 
-static int has_whitespace( const char *str)
-{
-	while( *str ) {
-		if( isspace(*str++) ) {
-			return( 1 );
-		}
-	}
-
-	return( 0 );
-}
-
 static void compress_path( MyString &path )
 {
 	char	*src, *dst;
@@ -1474,14 +1463,6 @@ int SubmitHash::SetStdFile( int which_file )
 			ABORT_AND_RETURN( 1 );
 		}
 	}
-	
-	if( has_whitespace(macro_value) ) 
-	{
-		push_error(stderr, "The '%s' takes exactly one argument (%s)\n", 
-				 generic_name, macro_value );
-		free(macro_value);
-		ABORT_AND_RETURN( 1 );
-	}	
 
 	MyString tmp = macro_value;
 	if ( check_and_universalize_path(tmp) != 0 ) {
-- 
2.13.5

